app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on the set up with devise but I think it would be great if you add : confirmable as well here since the requirements mentioned that you should Ask for confirmation by email. so kindly add it and after signing up the user should receive a link to ask for confirmation I think for now you could see the link in the terminal not in a real email address so please add it 💯🚀😄🔥


![image](https://github.com/OsvaldoBC/blog/assets/58553711/e7ffb033-0f23-4f52-a5cb-7aa95bd84c84)